inspirehep/modules/migrator/tasks.py

```diff
@@ -29,10 +29,12 @@
 import zlib
 from collections import Counter
 from itertools import chain
+from simplejson import loads
```

Comment:
why simplejson?

Reply:
I vaguely remember it being used somewhere else, no reason beyond that. Should I change it to `json`?